src/gobapi/graphql/schema.py

```diff
@@ -204,12 +206,15 @@ def get_graphene_query():
         attributes = {attr: graphene_type(value["type"], value["description"]) for attr, value in
                       collection["attributes"].items() if
                       not graphene_type(value["type"]) is None}
-        root_connection_fields[collection_name] = FilterConnectionField(root_connection_class, **attributes)
-        connection_fields[collection_name] = FilterConnectionField(rel_connection_class, **attributes)
+        root_connection_fields[f'{catalog_name}_{collection_name}'] = FilterConnectionField(root_connection_class,
+                                                                                            **attributes)
+        connection_fields[f'{catalog_name}_{collection_name}'] = FilterConnectionField(rel_connection_class,
+                                                                                       **attributes)
 
         # Use root_connection_class for inverse relations as well. No need for bronwaardes here.
-        inverse_connection_fields[collection_name] = FilterConnectionField(root_connection_class, **attributes)
-        base_models[collection_name] = base_model
+        inverse_connection_fields[f'{catalog_name}_{collection_name}'] = FilterConnectionField(root_connection_class,
```

Comment:
The combination is repeated over and over again.
Table name is encapsulated using model.get_table_name
Should you not encapsulate the collection name also?

Reply:
It's actually the same logic as gate_table_name, so I will use this function